site/specs-temp/numbers.json

```diff
@@ -20404,269 +20404,625 @@
           }
         }
       }
-    }
-  },
-  "components": {
-    "securitySchemes": {
-      "httpBasic": {
-        "type": "http",
-        "scheme": "basic"
-      }
     },
-    "schemas": {
-      "UcTrunkingSeats": {
-        "type": "object",
-        "properties": {
-          "UcTrunkingSeats": {
-            "type": "object",
-            "properties": {
-              "TotalCount": {
-                "type": "integer"
-              }
-            }
-          }
-        }
-      },
-      "AbstractOriginationSmsTnHistory": {
-        "type": "object",
-        "properties": {
-          "NPANXXDetailList": {
-            "type": "array",
-            "items": {
-              "type": "object",
-              "title": "NPANXXDetail",
-              "properties": {
-                "TerminationHost": {
-                  "type": "object",
-                  "properties": {
-                    "NPANXX": {
-                      "type": "string"
-                    },
-                    "Tier": {
-                      "type": "integer"
-                    },
-                    "Count": {
-                      "type": "integer"
-                    }
+    "/identitydomains/{domainName}/{roles}": {
```

Comment:
So looking at the response are we sure we want to expose this out to the public? The server side isn't filtering out roles that we do not mark as "CustomerVisible" which means that no roles would be non-customer visible if they are consuming the API. We would also be exposing which of our Roles are "Internally" scoped. 

We might want to get Infosec to weight in on some of the information that we are documenting and exposing to the customers.

Reply:
Hey @tmonck, I think backend filters out all roles user should not have access to. Depending on user privileges one will see both internal and external, another will see only external / owned by this user.

Reply:
<img width="971" alt="Screenshot 2022-10-05 at 16 15 32" src="https://user-images.githubusercontent.com/5918477/194069375-accb18c0-022b-4b3f-b502-0691cfdd0b18.png">